httpbakery/client_test.go

```diff
@@ -278,6 +278,16 @@ func (s *ClientSuite) TestMacaroonCookiePath(c *gc.C) {
 	assertCookieCount("/foo/bar/", 1)
 	assertCookieCount("/foo/bar/baz", 1)
 
+	cookiePath = "/foo"
```

Comment:
Because of the way the test works it can't be anything outside of the request path otherwise doRequest fails. but I've done "../bar".